packages/pegasus/src/contract.js

```diff
@@ -0,0 +1,34 @@
+import { prepareWhenableModule } from '@agoric/whenable';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { makePegasus } from './pegasus.js';
+
+import '@agoric/zoe/exported.js';
+
+import '../exported.js';
+
+/**
+ * @type {ContractStartFn<Pegasus, never, {}, {
+ *   board: ERef<BoardDepositFacet>,
+ *   namesByAddress: ERef<import('@agoric/vats').NameHub>
+ * }}
+ */
+export const start = (zcf, privateArgs, baggage) => {
+  const zone = makeDurableZone(baggage);
+
+  const whenZone = zone.subZone('when');
+  const { when } = prepareWhenableModule(whenZone);
+
+  const { board, namesByAddress } = privateArgs;
+
+  // start requires that the object passed in must be durable. Given that we
```

Comment:
I'm struggling to understand this. What object must be durable? help me find the docs for this constraint on `start()`? I don't think I'm familiar.

Reply:
This was an interesting problem that @michaelfig and I were trying to wrap our heads around. This was originally caught by multiple non-network/ibc vats, and we had the suspicion it was around the `upgradability: 'canUpgrade' meta. cc @michaelfig, who may be able to provide a better explanation. We weren't able to root cause _why_ it needed to be durable (since this is `start` vs `prepare`)

>  What object must be durable
In this case, the `publicFacet` needed to be durable. Before this change, it was just a `Far` object